telemetry/data_pipeline_mvp/grab_from_serial.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+"""
+objective - grab data from serial, parse, pass into redis timeseries 
+
+"""  
+
+import serial
+import time
+from redistimeseries.client import Client
+
+# figure out way to automatically detect port from python? 
+
+
+
+class Data_parser(object):
+    def __init__(self):
+        self.port_name = "/dev/ttyUSB0"
+        self.baudrate = 9600
+        self.read_frequency = 20 # in hz  
+        self.data_channels = ['arduino_data']
```

Comment:
Remove hard coded values, pass these in as inputs to the constructor. If you want to have default values, denote it in the constructor definition

```
def __init__(self, port_name='dev/ttyUSB0', baudrate=9600, read_frequency_hz=20, data_channels=['arduino_data']):
	self.port_name = port_name
... etc
```

Reply:
Might be worth having a `config.py` file that can be overwritten using OS env variables. This will be especially useful if/when we dockerize the platform.

Something like

```python
import os

PORT_NAME = os.getenv("TELEMETRY_PORT_NAME") or "/dev/ttyUSB0"
...
```

Reply:
I think there's a better way to do this with udev, but that's definitely not a now problem and should be documented in confluence. hmu when we revisit this closer to deployment time.